tools/biotransformer/.shed.yml

```diff
@@ -0,0 +1,10 @@
+categories:
+- Metabolomics
+description: BioTransformer is a tool for prediction of small molecule metabolism in mammals.
+homepage_url: https://gitlab.ics.muni.cz/recetox/mass-spectrometry/biotransformer
```

Comment:
I believe this should point to the official webpage of the software - i.e. we should probably use https://bitbucket.org/djoumbou/biotransformerjar from https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0324-5

Reply:
It's changed to the official webpage of the software in the recent commit.